packages/block-library/src/cover/edit.js

```diff
@@ -382,6 +375,12 @@ function CoverEdit( {
 		} );
 	};
 
+	const onUploadError = ( message ) => {
+		createErrorNotice( Array.isArray( message ) ? message[ 2 ] : message, {
```

Comment:
When can we end up with an array of `message`? It seems to me that we can't: https://github.com/WordPress/gutenberg/blob/trunk/packages/media-utils/src/utils/upload-media.js#L108.

Reply:
If the errors are coming from the REST API - https://github.com/WordPress/gutenberg/blob/fb3d84b4a8245cc25b5978eddf95e4a3ff8d7477/packages/media-utils/src/utils/upload-media.js#L207-L211.

Reply:
Yeah, it's kind of weird to convert the `string` message to array :)